lib/vector-buffers/src/test/common/helpers.rs

```diff
@@ -44,7 +44,8 @@ where
     F: FnOnce(&Path) -> Fut,
     Fut: Future<Output = V>,
 {
-    let buf_dir = TempDir::new().expect("creating temp dir should never fail");
+    let buf_dir =
+        TempDir::with_prefix("vector-buffers").expect("creating temp dir should never fail");
```

Comment:
This is fine since it was already like this, but I can imagine that this _could_ fail due to permissions or file system issues. Perhaps it'd be more accurate to say "cannot recover from failure to create temp dir".